docs/VISION.md

```diff
@@ -0,0 +1,23 @@
+# OpenTelemetry Service Long-term Vision
+
+The following are high-level items that define our long-term vision for OpenTelemetry Service, what we aspire to achieve. This vision is our daily guidance when we design new features and make changes to the Service.
+
+This is a living document that is expected to evolve over time.
+
+## Performant
+Highly stable and performant under varying loads. Well-behaved under extreme load, with predictable, low resource consumption.
+
+## Observable
+Expose own operational metrics in a clear way. Be an exemplar of observable service. Allow configuring the level of observability (more or less metrics, traces, logs, etc reported).
+
+## Multi-Data
+Support traces, metrics, logs and other relevant data types.
+
+## Usable Out of the Box
+Reasonable default configuration, supports popular protocols, runs and collects out of the box.
+
+## Extensible
+Extensible and customizable without touching the core code. Can create custom agents based on the core and extend with own components. Welcoming 3rd party contribution policy.
+
+## Unified Codebase
```

Comment:
Suggestion: you can also mention unified configuration for Agent and Collector.

Reply:
I tried to keep this doc very short and high-level only. The configuration unification is part of the general unification vision, so I think we can omit it in this high-level doc.

Reply:
Makes sense to me.